### PR TITLE
Minor formatting

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md [hunk 1]
```diff
@@ -33,8 +33,7 @@ When you configure a CloudWatch source, you have the option of defining *AWS tag
 Tag filtering is only supported for user-defined AWS tags, not for AWS-generated tags.
 :::
 
-Here’s how tag filtering works:
-
+Here’s how tag filtering works.
```
Comment (stacykor) [inline docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md:36]:
```suggestion
Here’s how tag filtering works:
```

File: docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md [hunk 2]
```diff
@@ -33,8 +33,7 @@ When you configure a CloudWatch source, you have the option of defining *AWS tag
...
 * If you don't specify a tag filter for a namespace, the source will collect all metrics for the namespace.
```
Comment (stacykor) [inline docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md:37]:
```suggestion
*Specify a tag filter for a namespace **OR** else the source will collect **all** metrics for the namespace.
```

Comment (stacykor) [reply docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md:37]: Given the severity of the impact, make what they should do a command, and tell them why.

**Review by stacykor — CHANGES_REQUESTED** [review]

**Review by stacykor — APPROVED** [review]

**Comment by kimsauce** [comment]
Closing - this was a test PR

**Pull request closed without merging**